posawesome/public/js/posapp/components/pos/ClosingDialog.vue

```
@@ -3,125 +3,154 @@
                     {{ currencySymbol(pos_profile.currency) }}
-                    {{ formatCurrency(item.opening_amount) }}</template>
+                    {{ formatCurrency(item.returns) }}
+                  </template>
+
+                  <!-- Editable Closing Amount -->
+                  <template v-slot:item.closing_amount="{ item }">
+                    <v-text-field
+                      v-model.number="item.closing_amount"
+                      type="number"
+                      label="Edit Closing Amount"
+                      @input="updateDifference(item)"
```

ESLint

[error] VTextField: @input has been replaced with @update:modelValue

(vuetify/no-deprecated-events)